extensions/wikia/PortabilityDashboard/PortabilityDashboard.i18n.php

```diff
@@ -25,11 +25,13 @@
 	'portability-dashboard-infoboxes-desc' => 'Number of non-portable infoboxes on this community with link to Special:Insights',
 	'portability-dashboard-impact-header' => 'Migration Impact',
 	'portability-dashboard-impact-desc' => 'Number of this community\'s content page views coming from pages with non-portable infoboxes',
-	'portability-dashboard-language-filter-label' => 'Filter table by language',
+	'portability-dashboard-language-filter-label' => 'Or filter table by language',
```

Comment:
The contextual binding between 'portability-dashboard-search-headline' and 'portability-dashboard-language-filter-label' is not reflected in the way markup is structured. While not a problem for EN only page, it's worth remembering that this is in general considered a bad practice - https://www.mediawiki.org/wiki/Localisation